### DISCUSSION TITLE
API mit TypeScript und Nestjs

### DISCUSSION THREAD

**Discussion by sarina-nikolic** [neuefische/questions#112] [Web] [answered]
Hi Leute, ich tüftel gerade ein wenig an einer http API mit TypeScript und Nestjs herum und stehe gerade auf dem Schlauch. Ich brauche keine echte Datenbank, sondern hab nur eine kleine Objektliste. Die Endpunkte hab ich erstellt und konkret komme ich jetzt nicht weiter, wie ich nun dieses Objekt ansprechen soll. Vielleicht mag jemand mal rüber gucken. Vielen lieben Dank schon mal. :innocent: https://github.com/sarina-nikolic/snackbox

**Comment by mpagels** [comment, marked as answer]
Ich hatte eine Fehlermeldung, nachdem ich dein Project ausgecheckt und mit `npm start` gestartet habe.

Irgendwie sowas: `nestjs Cannot find module 'dist/main'` 

Das Problem habe ich gelöst, in dem ich alle Dateien in dem  `src/snacks` Order in den `src` Ordner verschoben habe.
Und dann ging `npm start`.

Dann habe ich in `snacks.service.ts` in Zeile 12 einfach mal das array aus `snacks-data.ts` eingefügt. Und dann kannst du per `http:localhost:3000/snacks` deine snacks anzeigen lassen. Hilft dir das für den Anfang?

**Comment by sarina-nikolic** [comment]
Manchmal weiß man gar nicht, wie dicht man schon dran war. :D Lieben Dank.
